### ISSUE TITLE
[Bug] Effect - Show Text - Random Preset Location doesn't work

### ISSUE THREAD

**Issue by SolveSaint** [crowbartools/Firebot#2829]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When adding the effect Show Text with Random Preset Location selected, then effect doesn't work.

### Expected Behavior

The text should be shown in a random location within the selected sizing bounds.

### Steps To Reproduce

Trigger on Chat Message (Twitch)
Effects > Show Text > Random Preset Location
The effect will not work.

### Version

5.63.2

### What operating system are you using Firebot on?

Windows 11

**Comment by CaveMobster** [comment]
Good catch, thank you for reporting it! I fixed the issue, it will be part of the next release.
